models/MonthlyProfile.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "MonthlyProfile",
+  "type": "object",
+  "description": "Prêt ou plan progressif variant mensuellement.",
+  "properties": {
+    "type": {
+      "type": "string",
+      "description": "Fréquence des variations.",
```

Comment:
Description of this field looks weird

Reply:
Let's use the generic "Type de profil de remboursement."